src/main/java/sg/ncl/MainController.java

```diff
@@ -4999,7 +5039,7 @@ private Realization getCleanRealization() {
         }
 
         String result = respEntity.getBody().toString();
-        log.info(result);
+        // log.info(result);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)